Assets/HoloToolkit/Input/Scripts/InputEventData/InputEventData.cs

```diff
@@ -2,21 +2,28 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine.EventSystems;
+using UnityEngine.VR.WSA.Input;
```

Comment:
This breaks cross platform support because the code uses "InteractionPressKind pressKind".
This is not defined in the Windows Runtime 10.0.15063.0 file header Windows.UI.Input.Spatial.h

The header has a define "Windows::UI::Input::Spatial::SpatialInteractionPressKind" and elsewhere there is the entry "Windows.UI.Input.Spatial.SpatialInteractionPressKind".
The documentation for the API at 
https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.spatial.spatialinteractionpresskind
reflect this.
When trying to replace the code 
with the code "Windows.UI.Input.Spatial.SpatialInteractionPressKind pressKind"
The problem is that nothing matches the headers defined in [Windows.UI.Input.Spatial.h](https://github.com/Microsoft/cppwinrt/blob/master/10.0.15063.0/winrt/Windows.UI.Input.Spatial.h) and therefore nobody will be able to write Spatial mapping code in C++ or C# or DirectX, because the old Holographic code for HoloLens, wont match the code for Windows Mixed Reality.
The Toolkit will ALWAYS require Unity 5.6.* and when we move to the new Unity 2017.1 and Visual Studio 2017 there will be a massive code break.

The other problem with this adding of Unity code is that "InteractionManager.SourceEventArgs "
mismatches the 15063 runtime documentation
https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.spatial.spatialinteractionsourceeventargs
"SpatialInteractionSourceEventArgs Class", even the design of the name is a conflict,
to reflect the C++ WinRT, "Interaction.SourceEventArgs"

Why doesn't all the code work the [Windows.UI.Input.Spatial Namespace 
](https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.spatial)

The alternative is that developers will write code in 10.0.15063 C++ standalone leaving Unity stranded